src/components/Modal/ModalPropTypes.js

```diff
@@ -23,7 +23,9 @@ const propTypes = {
         CONST.MODAL.MODAL_TYPE.CONFIRM,
         CONST.MODAL.MODAL_TYPE.CENTERED,
         CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED,
-        CONST.MODAL.MODAL_TYPE.POPOVER,
+        CONST.MODAL.MODAL_TYPE.POPOVER_CENTER_BOTTOM,
```

Comment:
Maybe it would be easier to do something like `type: PropTypes.oneOf(_.values(CONST.MODAL.MODAL_TYPE))`?